antlr_tsql/tsql.g4

```diff
@@ -1040,7 +1068,7 @@ next_value_for_function
 // https://msdn.microsoft.com/en-us/library/ms189798.aspx
 ranking_windowed_function
     : (RANK | DENSE_RANK | ROW_NUMBER) '(' ')' over_clause
-    | (NTILE | FIRST_VALUE | LEAD | LAG) '(' expression ')' over_clause
+    | (NTILE | FIRST_VALUE | LEAD | LAG) '(' expression (',' expression)* ')' over_clause
```

Comment:
`expression` now becomes a list, if this is used elsewhere, it can break SCTs, so we would have to check that.

Reply:
Isn't the list optional such that `expression` is only a list when there are two or more elements, or is now a list with one element or more every time?
And if so, how should I test for it?

Reply:
Solved